docs/.vuepress/theme/styles/custom/base/_type-and-content.scss

```
@@ -124,6 +124,15 @@ div[class~='language-yaml']:before {
 // and there is no reason to do so
 pre div.code-copy {
   height: 0;
+  .hover{
+    // Used important as it has to override the inline style from vuepress-plugin-code-copy
+    bottom: 0px !important;
+
+    // Used to fix the overlap of the scroll bar as due to the
+    // above property and hence we set it manually below
+    margin-bottom: 25px !important;
+    margin-right: 15px !important;
+  }
```

Please remove the unit from the `0px` value and the `!important` from the margin property declarations as it’s not needed there.

```suggestion
  .hover {
    // Used important as it has to override the inline style from vuepress-plugin-code-copy
    bottom: 0 !important;

    // Used to fix the overlap of the scroll bar as due to the
    // above property and hence we set it manually below
    margin-bottom: 25px;
    margin-right: 15px;
  }
```

---

Done dona done ✅